src/lib/slider/slider.ts

```diff
@@ -247,6 +247,13 @@ export class MatSlider extends _MatSliderMixinBase
   }
   private _value: number | null = null;
 
+  /**
+   * Function that will be used to format the value before it is displayed
+   * in the thumb label. Can be used to format very large number in order
+   * for them to fit into the slider thumb.
+   */
+  @Input() displayWith: (value: number | null) => string | number;
```

Comment:
`displayWith` is kind of a weird name. What about `formatter` or `formatFunction`?

Reply:
I went with this one, because we use the same for the autocomplete.